packages/validator/__tests__/validator.test.js

```diff
@@ -0,0 +1,797 @@
+'use strict';
+
+const validator = require('../lib/validator');
+
+const specs1 = `{
```

Comment:
Renamed and moved in 62eb558. This is something I should have addressed sooner, at first I thought there would be just a few tests so it would be fine, but it quickly got out of hand... 
Each specification is now right above the expect it is used in, so there is minimal scrolling required to view them. 
I don't think one spec per test here is a good idea. For example in `missing parameters` test, there are multiple specs which are testing the same thing, however since requirements for parameters might be influenced by many things, there are multiple similar inputs, the only way to differentiate them would be write out the errors they should produce, but in that case I would much rather stick with the plain numbers in the same test. Other than that the specifications have a meaningful name, not the previous specs1, 2, 3...

Reply:
Alright that's fine. I think the renaming already looks much better 👍🏻